playback/src/player.rs

```diff
@@ -402,7 +426,37 @@ impl PlayerInternal {
         }
     }
 
-    fn load_track(&self, track_id: SpotifyId, position: i64) -> Option<Decoder> {
+    fn parse_normalisation<T: Read + Seek>(&self, file: &mut AudioDecrypt<T>) -> NormalisationConfig {
+        let track_gain_db: f32;
+        let track_peak: f32;
+        let album_gain_db: f32;
+        let album_peak: f32;
+
+        file.seek(SeekFrom::Start(144)).unwrap();
+        track_gain_db = file.read_f32::<LittleEndian>().unwrap();
+        debug!("Track gain: {}db", track_gain_db);
```

Comment:
Use a single debug! line for all 4 values (eg `debug!("Normalization: {:?}", data);`), potentially in `load_track`.